R/contingencytables.R

```diff
@@ -1032,19 +1036,19 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
   return(list(rows = tests.rows, rownames = row.rownames))
 }
 
-.crossTabOddsRatioRows <- function(analysisContainer, var.name, groupList, options, ready) {
+.crossTabOddsRatioNormalRows <- function(analysisContainer, var.name, groupList, options, ready) {
```

Comment:
```suggestion
.crossTabOddsRatioRows <- function(analysisContainer, var.name, groupList, options, ready) {
```
what's the benefit of "Normal" here?